docs/sdk/examples/react.html

```diff
@@ -0,0 +1,319 @@
+					font-family: "SF Mono",menlo,monaco,"Roboto Mono",Consolas,"Lucida Console",monospace;
+					font-size: .866666em;
+					padding: 1.333em;
+					background: #2b2c26;
+					color: #fff;
+					margin-bottom: 10px;
+				}
+
+				ul.event-log__events {
+					list-style-type: none;
+					padding: 0;
+					margin: 0;
+					max-height: 200px;
+					overflow: auto;
+				}
+
+				/* 3rd sample styling. */
+				.editor-preview {
+					float: right;
+					width: 49%;
+					border: 1px solid #d1d1d1;
+					padding: 10px;
+				}
+
+				.editor-preview h2 {
+					padding-top: 0;
+				}
+
+				@media screen and (max-width: 768px) {
+					.editor-preview {
+						width: 100%;
+					}
+				}
+			</style>
+
+			<h1>React Integration <a class="documentation" href="https://ckeditor.com/docs/ckeditor4/latest/guide/dev_react.html">Documentation</a></h1>
+
+			<p>
+				<strong>React integration</strong> allows you to easily use CKEditor 4 as a <a href="https://reactjs.org/">React</a> component, using <code>&lt;CKEditor /&gt;</code> JSX tag.
+			</p>
+
+			<p>
+				Following examples showcase the most important features of React integration.
+			</p>
+
+			<p>
+				<strong>Click and change an example:</strong>
+			</p>
+
+			<div id="app"></div>
+			<script src="assets/react/main.js"></script>
+		</section>
+	</section>
+
+	<footer class="sdk-footer">
+	</footer>
+
+<script type="text/jsx" data-sample="1" data-sample-preserve-whitespace data-sample-strip-outer-tag data-sample-template="empty" data-sample-highlighter="jsx">
+import ReactDOM from 'react-dom';
+import CKEditor from 'ckeditor4-react';
+
+ReactDOM.render(
+	<CKEditor data="<p>This is CKEditor 4 instance created by ️⚛️ React.</p>" />,
+	document.getElementById( 'app' )
+);
+</script>
+<script type="text/jsx" data-sample="2" data-sample-preserve-whitespace data-sample-strip-outer-tag data-sample-template="empty" data-sample-highlighter="jsx">
+import ReactDOM from 'react-dom';
+import CKEditor from 'ckeditor4-react';
+
+ReactDOM.render(
+	<CKEditor type="inline" data="<p>This is CKEditor 4 instance created by ️⚛️ React.</p>" />,
+	document.getElementById( 'app' )
+);
+</script>
+<script type="text/jsx" data-sample="3" data-sample-preserve-whitespace data-sample-strip-outer-tag data-sample-template="empty" data-sample-highlighter="jsx">
+import ReactDOM from 'react-dom';
+import CKEditor from 'ckeditor4-react';
+
+ReactDOM.render(
+	<CKEditor
+		data="I'm CKEditor 4 instance."
+		config={{
+			toolbar: [
+				[ 'Bold', 'Italic' ],
+				[ 'Cut', 'Copy' ],
+				[ 'About' ]
+			]
+		}}
+		onFocus={customHandler}
+		onBlur={customHandler}
+		onChage={customHandler}
+		onSelectionChange={customHandler}
+	/>,
+	document.getElementById( 'app' )
+);
+
+function customHandler( evt ) {
+	console.log( evt );
+}
+</script>
+<script type="text/jsx" data-sample="4" data-sample-preserve-whitespace data-sample-strip-outer-tag data-sample-template="empty" data-sample-highlighter="jsx">
+import React from 'react';
```

Comment:
This sample will fail to compile, Component class is not defined.

Should be `import React, { Component } from 'react';`.